scipy/special/meson.build

```diff
@@ -133,12 +133,12 @@ cython_special_pxd = fs.copyfile('cython_special.pxd')
 uf_cython_gen = generator(cython,
   arguments : cython_args,
   output : '@BASENAME@.c',
-  depends : [_cython_tree, _ufuncs_pxi_pxd_sources, cython_special_pxd])
+  depends : [_cython_tree, _ufuncs_pxi_pxd_sources, cython_special_pxd, cython_special[0]])
```

Comment:
I can reproduce the CI `meson` typing complaint locally if I downgrade to the `meson` `1.4.0` used by the Windows Intel job. Do you remember if there's a specific need for that pin? IIRC these oneAPI CI additions are new enough that it is probably fresh in someone's mind.

Reply:
There wasn't a specific reason for this I believe.